cql3/Cql.g

```diff
@@ -1897,10 +1952,13 @@ unreserved_function_keyword returns [sstring str]
 basic_unreserved_keyword returns [sstring str]
     : k=( K_KEYS
         | K_AS
+        | K_CLUSTER
         | K_CLUSTERING
         | K_COMPACT
         | K_STORAGE
```

Comment:
Shouldn't DESCRIBE also be an unreserved keyword? Or we break previously valid schemas that had DESCRIBE in them.

Reply:
Yes, you're right. added DESCRIBE and DESC to unreserved keywords